packages/token-vault/src/lib/client.ts

```diff
@@ -45,19 +67,25 @@ export function client(config: ClientConfig) {
 
       return await registerServiceWorker();
```

Comment:
It's just semantics I suppose, but yes.

I think we can have an `await` on its own line if we are managing the failure of the promise here in this function with a try catch as the `return <my-promise>` will force the error to bubble up to the next layer in the stack, where as breaking it into two lines would have it error within this function.

Since we are not managing any failure here we could just `return registerServiceWorker()` and it should behave the same.